rskj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -180,6 +181,18 @@ public boolean init() {
             return false;
         }
 
+        // Prevent transactions with excessive address size
+        byte[] receiveAddress = tx.getReceiveAddress();
+        if (receiveAddress != null && receiveAddress.length > MAX_ADDRESS_LENGTH) {
+            if (logger.isWarnEnabled()) {
```

Comment:
We need to check if it's warn enabled? Shouldn't be enough with the log level config?

Reply:
Done https://github.com/rsksmart/rskj/pull/304/commits/19002c019392387e53dad0b31262ef99d6e4759c